### PR TITLE
Square Kirby command

### PR DESCRIPTION
**Pull request by jaxxibae**
+ Suggested by @pedrofracassi on Discord. [Blame Pedro](https://i.imgur.com/FgBSPs5.png).

This image gives me nightmares.

### CODE REVIEW

**Comment by jaxxibae** [comment]
This isn't RFR yet, I need to distort the image 1st.

**Comment by jaxxibae** [comment]
I beleive it's RFR now.

**Comment by metehus** [comment]
image
pls

**Comment by jaxxibae** [comment]
If you wanna help, please do. My mind is still hurting with that.

File: src/commands/image-manipulation/squarekirby.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          acceptBot: true
+        }
+      }]
+    }, client)
+  }
+
+  async run ({ channel }, image) {
+    channel.startTyping()
+    const squareKirby = await CanvasTemplates.squareKirby(image)
+    channel.send(new MessageAttachment(squareKirby, 'squarekirby.jpg'))
```
Comment (Doges) [inline src/commands/image-manipulation/squarekirby.js:25]: ".then(() => channel.stopTyping(true))" thanks

**Review by Doges — CHANGES_REQUESTED** [review]

File: src/commands/image-manipulation/squarekirby.js (hunk 1 above)
Comment (pedrofracassi) [reply src/commands/image-manipulation/squarekirby.js:25]: make it a suggestion

**Comment by jaxxibae** [comment]
Closing this PR, since the meme has already died.

**Pull request closed without merging**